nncf/onnx/hardware/fused_patterns.py

```diff
@@ -256,6 +275,16 @@ def create_linear_batch_norm_activations() -> GraphPattern:
     return linear_batch_norm
 
 
+@ONNX_HW_FUSED_PATTERNS.register(HWFusedPatternNames.LINEAR_BATCH_NORM_ACTIVATIONS_ARITHMETIC)
```

Comment:
It is not clear for me why this fused pattern was added. Could you please briefly explain the reason?

Reply:
Sure, I incorrectly didn't add a Div layer in HSwish pattern. If HSwish pattern has Div op, there is no need in `LINEAR_BATCH_NORM_ACTIVATIONS_ARITHMETIC` pattern